sql/bigconfig.yml

```diff
@@ -8,9 +8,9 @@ saved_metric_definitions:
       threshold:
         type: CONSTANT
         upper_bound: 0
-      metric_schedule:
-        named_schedule:
-          name: default
+      schedule_frequency:
+        interval_type: MINUTES
+        interval_value: 0
```

Comment:
There is an open Bigeye bug for this. It should work in theory, but in practice it will break deploys

Reply:
Should I just revert it back to default or just remove the scheduling completely?

Reply:
just revert it back for now. Not defining a schedule will default to a 24h schedule frequency